packages/de1/src/converters/shot.ts

```diff
@@ -27,9 +27,7 @@ function decode(data: Buffer): Shot {
     .short("groupPressure", v => v / 4096)
     .short("groupFlow", v => v / 4096)
     .short("mixTemp", v => v / 256)
-    .char("headTemp1")
-    .char("headTemp2")
-    .char("headTemp3")
+    .bytes("headTemp", 3, convert_3_char_to_U24P16)
```

Comment:
```suggestion
    .bytes("headTemp", 3, convert3CharToU24P16)
```

I would camelCase this, because these are too: https://github.com/LukasBombach/de1/blob/master/packages/de1/src/converters/shotFrame.ts#L26

Reply:
Sure, was using the original Tcl name to make it easier to connect them